### PR TITLE
Feat/sbt update limit

### PR DESCRIPTION
**Pull request by zhangjing28**
## Description

* close: #757 #769 #770 #771 #772
*  update the upload min limit to 3

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [ ] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [X] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [ ] Update the version number in `package.json` and `src/config/common.json`
- [ ] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [ ] Update `CHANGELOG.md`
- [ ] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -68,6 +68,10 @@ input[type='number']::-webkit-outer-spin-button {
   font-family: 'Red Hat Mono' !important;
 }
 
+.font-red-hat-text {
```
Comment (jincan39) [inline src/styles/index.scss:71]: think it's no need? by default all the font is 'Red Hat Text'

**Review by jincan39 — APPROVED** [review]
LGTM

File: src/styles/index.scss (hunk 1 above)
Comment (zhangjing28) [reply src/styles/index.scss]: In this `src/styles/index.scss`, it's only set `html,.ui,h1,h2,h3,h4,h5,h6,textarea,view,div` these tags `font-family`, not include `p, button, input`😂. I already updated that

File: src/pages/SBTPage/components/Home/index.tsx [hunk 2]
```diff
@@ -198,7 +198,7 @@ const Home = () => {
       </div>
 
       {/* <div className="mt-4 w-full">
```
Comment (Kevingislason) [inline src/pages/SBTPage/components/Home/index.tsx:200]: can this commented code be deleted?

**Review by Kevingislason — APPROVED** [review]

File: src/pages/SBTPage/components/Home/index.tsx (hunk 2 above)
Comment (zhangjing28) [reply src/pages/SBTPage/components/Home/index.tsx:200]: We need this FAQ part in the next version, so I think we can keep it by now, and will remove the annotation in the next Version

**Review by jincan39 — APPROVED** [review]
LGTM

**Pull request merged**